easybuild/easyconfigs/m/MultiQC/MultiQC-1.12-foss-2021b.eb

```
@@ -0,0 +1,72 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+# Elements derived from work by Pablo Escobar
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+# Note that Click in Python 3 requires that you change your locale to unicode before invoking your Python script.
+# See: https://click.palletsprojects.com/en/7.x/python3/
+
+easyblock = 'PythonBundle'
+
+name = 'MultiQC'
+version = '1.12'
+
+homepage = 'https://multiqc.info'
+description = """Aggregate results from bioinformatics analyses across many samples into a single report.
+
+ MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
```

Bit of a nitpick, but this reads oddly, and I think should be:
```suggestion
 MultiQC searches a given directory for analysis logs and compiles an HTML report. It's a general
```

---

@SethosII yes, I assumed you copied it from the upstream site. 
My preference would be to change it, but I would completely understand if you prefer to stick to their exact text.